boards/FSM/Src/App/App_SetPeriodicCanSignals.c

```diff
@@ -27,16 +26,16 @@ void App_SetPeriodicSignals_WheelSpeedInRangeChecks(const struct FsmWorld *world
     App_SetPeriodicCanSignals_InRangeCheck(
         can_tx, left_wheel_speed_in_range_check, App_CanTx_SetPeriodicSignal_LEFT_WHEEL_SPEED,
         App_CanTx_SetPeriodicSignal_LEFT_WHEEL_SPEED_OUT_OF_RANGE,
-        CANMSGS_FSM_NON_CRITICAL_ERRORS_LEFT_WHEEL_SPEED_OUT_OF_RANGE_OK_CHOICE,
-        CANMSGS_FSM_NON_CRITICAL_ERRORS_LEFT_WHEEL_SPEED_OUT_OF_RANGE_UNDERFLOW_CHOICE,
-        CANMSGS_FSM_NON_CRITICAL_ERRORS_LEFT_WHEEL_SPEED_OUT_OF_RANGE_OVERFLOW_CHOICE);
+        CANMSGS_FSM_WARNINGS_LEFT_WHEEL_SPEED_OUT_OF_RANGE_OK_CHOICE,
```

Comment:
```suggestion
        CANMSGS_FSM_WARNING_LEFT_WHEEL_SPEED_OUT_OF_RANGE_OK_CHOICE,
```
I personally think this sounds better. It now sounds like "FSM Warning: Left Wheel Speed Out of Range!"

but I see why it was written like this, as in it falls under the warnings 'group'

Reply:
if you decide to change please change the rest to match

Reply:
I looked into this quite a bit, and while I agree it looks nice, I don't think its worth what it'd cost to achieve. In order to get the codegen to generate the can message choices like you have above, the message 'groups' in the .dbc would need to be changed to "FSM_WARNING" instead of "FSM_WARNINGS". 
```
BO_ 300 FSM_WARNING: 8 FSM
...
SG_ LEFT_WHEEL_SPEED_OUT_OF_RANGE: 9|2@1+ (1,0) [0|2] "" DEBUG
SG_ RIGHT_WHEEL_SPEED_OUT_OF_RANGE: 11|2@1+ (1,0) [0|2] ""  DEBUG
...
```

All that being said, the actual calls to send the messages in the code would not change, as they do not include "warning" in them IE:
 ```
App_CanTx_SetPeriodicSignal_LEFT_WHEEL_SPEED_OUT_OF_RANGE(...)
```
Only the generated choices would change:
```
#define CANMSGS_FSM_WARNING_LEFT_WHEEL_SPEED_OUT_OF_RANGE_OK_CHOICE (0u)
#define CANMSGS_FSM_WARNING_LEFT_WHEEL_SPEED_OUT_OF_RANGE_UNDERFLOW_CHOICE (1u)
#define CANMSGS_FSM_WARNING_LEFT_WHEEL_SPEED_OUT_OF_RANGE_OVERFLOW_CHOICE (2u)
```

If we are okay with re-naming the groups to "_WARNINGS" then it's easily accomplished, in fact I've already pushed the required changes, but I feel like we should revert and leave it as "_WARNINGS"

Reply:
I agree with Will here, I think keeping it "_WARNINGS" makes sense because that comes from the CAN message name, which is the set of warnings, not a single warning. 
But I don't mind either way, up to you guys.